use-timescale/extensions/pgvector.md

```diff
@@ -0,0 +1,478 @@
+---
+title: pgvector PostgreSQL extension
+excerpt: Use the pgvector extension with your Timescale service
+products: [cloud]
+keywords: [services, settings, extensions, pgvector]
+tags: [extensions, pgvector]
+---
+
+# The `pgvector` extension
+
+The `pgvector` PostgreSQL extension helps you to store and search over machine
+learning-generated embeddings. It provides different capabilities that allows
+you to identify both exact and approximate nearest neighbors. It is designed to
+work seamlessly with other PostgreSQL features, including indexing and querying.
+
+For more information about these functions and the options available, see the
+[pgvector][pgvector-repo] repository.
+
+## Use the `pgvector` extension to create a `chatbot`
+
+The `pgvector` PostgreSQL extension allows you to create, store, and query
+OpenAI [vector embeddings][vector-embeddings] in Timescale. Learn how to use
+[Retrieval Augmented Generation (RAG)][rag-docs] to create a `chatbot` that combines
+your data with the power of ChatGPT using OpenAI and pgvector. RAG addresses the
+problem that a foundational model such as GPT-3 or GPT-4 may be missing some
+information needed to give a good answer because that information was not in the
+dataset used to train the model. For example, the information is stored in
+private documents or only became available recently.
+
+### Prerequisites
+
+Before you begin, make sure you have:
+
+*   Installed Python.
+*   Created a [Timescale][cloud-login] service.
+*   Downloaded the cheatsheet when you created the service. This sheet contains
+    the connection details for the database you want to use as a vector database.
+*   Signed up for an [OpenAI developer account][openai-signup].
+*   Created and API key and made a note of your OpenAI [API key][api-key].
+    <Highlight type="note">
+    If you are on a free plan there may be rate limiting for your API requests.
+    </Highlight>
```

Comment:
This won't work without whitespace on either side. Can we move it under the pre-reqs list, and specify that we're talking about the openAI account?

Reply:
<img width="774" alt="Screenshot 2023-06-29 at 4 49 25 PM" src="https://github.com/timescale/docs/assets/37059749/29ab6fbe-4ae9-447f-ab7f-da6594c689e6">

This is what I see in the preview.

Reply:
Ha! Well, I'm constantly surprised by this toolchain 😂 

Either way, maybe move it to be the last dot point, so we don't have that dangling entry?

Reply:
Done!!!